docs/adr/0015-security-fixes-non-wmde-software.md

```diff
@@ -0,0 +1,75 @@
+# 15) Including and Publishing Security fixes {#adr_0015}
+
+Date: 2021-05-03
+
+## Status
+
+proposed
+
+## Context
+
+WMDE needs to decide on a process of including security fixes of non-WMDE maintained software (e.g. Mediawiki, Elasticearch, WDQS) as well as WMDE maintained software (Wikibase, WDQS-frontend) and how to publish security fix releases of Wikibase suite in such cases.
+
+Security fixes will fall under the maintenance category and will most likely be worked on by campsite, however as the running of the pipeline and publishing of docker images requires special privileges it could make sense that this step is tasked to a certain group of people or that all engineers of WMDE are given the rights to publish releases.
+
+## List of steps reporting, fixing and releasing security updates
+
+### 1. Recognizing / Reporting
+
+- Report using `security@wikimedia.org` would be forwarded to WMDE staff
+- Report using [phabricator task](https://phabricator.wikimedia.org/maniphest/task/edit/form/75/)
+- Task will only be visible to security team at first then delegated to WMDE staff.
+
+### 2. Resolving
+
+- Don't publish the solution before it's deployed in WMF production environment
+  - Dont use public repositories
```

Comment:
What exactly does this mean? When we build the release, the patches will probably be in master already?

Reply:
I guess this stems from my uncertainty of what this task is supposed to include.

There were about 12 AC:s and I thought the ask was to cover the whole process.

Reply:
Sorry @toban ! I believe this line could go, as the one above "Don't publish the fix before it's deployed in WMF production environment" includes it.

Reply:
no problemo, done